SPECS/kubevirt/kubevirt.spec

```diff
@@ -0,0 +1,1670 @@
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+Summary:        Container native virtualization
+Name:           kubevirt
+Version:        0.51.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Management
+URL:            https://github.com/kubevirt/kubevirt
+#Source0:       https://github.com/kubevirt/kubevirt/archive/refs/tags/v0.51.0.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Source1:        disks-images-provider.yaml
+%global debug_package %{nil}
+BuildRequires:  glibc-devel
+BuildRequires:  golang
+BuildRequires:  golang-packaging
+BuildRequires:  pkgconfig
+BuildRequires:  rsync
+BuildRequires:  sed
+BuildRequires:  pkgconfig(libvirt)
+ExclusiveArch:  x86_64 aarch64
+
+%description
+Kubevirt is a virtual machine management add-on for Kubernetes
+
+%package        virtctl
+Summary:        Client for managing kubevirt
+Group:          System/Packages
+
+%description    virtctl
+The virtctl client is a command-line utility for managing container native virtualization resources
+
+%package        virt-api
+Summary:        Kubevirt API server
+Group:          System/Packages
+
+%description    virt-api
+The virt-api package provides the kubernetes API extension for kubevirt
+
+%package        container-disk
+Summary:        Container disk for kubevirt
+Group:          System/Packages
+
+%description    container-disk
+The containter-disk package provides a container disk functionality for kubevirt
+
+%package        virt-controller
+Summary:        Controller for kubevirt
+Group:          System/Packages
+
+%description    virt-controller
+The virt-controller package provides a controller for kubevirt
+
+%package        virt-handler
+Summary:        Handler component for kubevirt
+Group:          System/Packages
+
+%description    virt-handler
+The virt-handler package provides a handler for kubevirt
+
+%package        virt-launcher
+Summary:        Launcher component for kubevirt
+Group:          System/Packages
+
+%description    virt-launcher
+The virt-launcher package provides a launcher for kubevirt
+
+%package        virt-operator
+Summary:        Operator component for kubevirt
+Group:          System/Packages
+
+%description    virt-operator
+The virt-opertor package provides an operator for kubevirt CRD
+
+%package        tests
+Summary:        Kubevirt functional tests
+Group:          System/Packages
+
+%description    tests
+The package provides Kubevirt end-to-end tests.
+
+%prep
+%autosetup -p1
+
+%build
+export GOFLAGS="-buildmode=pie"
```

Comment:
We may want to have some default flags set in the future (if we don't already have them). Let's rewrite this, to not discard them.

Reply:
This is coming from the openSUSE spec file that was used. The rpm build (and the rpm itself) works without this. Where can I check what the default flags for golang are?